contrib/util/reset-ps-all-apps.sh

```diff
@@ -0,0 +1,12 @@
+apps=$(curl -H "Authorization: token $DEIS_TOKEN" http://$DEIS_SERVER/v1/apps  | jq -r '.results | map(.id) | join(" ")')
+
+for app in $apps; do
+  echo "Resetting containers of $app"
+
+  originalscale=$(curl -H "Authorization: token $DEIS_TOKEN" http://$DEIS_SERVER/v1/apps/$app/containers/ 2>/dev/null | jq -r '(.results) | [group_by(.type)[] | max_by(.num)] | [map(.type), map(.num)] | transpose | map([.[0], .[1] | tostring] | join("=")) | join(" ")')
```

Comment:
Again that format is not suitable to change to "typeA=N typeB=N". Probably doable with a combination of sed, awk and special magic, but not something doable with my capabilities.